tasks/functions/functions.js

```diff
@@ -15,31 +15,48 @@ function add(a, b) {
  * }
  */
 function getFullName(object) {
-
+    return object.firstName + ' ' + object.lastName;
 }
 
+
 /**
  * write fuction that checks is number is odd
  * true if odd, false if even
  */
 function isOdd(n) {
-
+    let x = n % 2;
+    if (x == 0) {
```

Comment:
you can put the statement here:
if ( n % 2 == 0) {...}

Reply:
thanks, fixed